joplin/css/editor.scss

```diff
@@ -51,3 +51,11 @@ footer {
 .editor__save-button {
   min-width: 130px;
 }
+
```

Comment:
hey @briaguya a couple things
1) I'm not sure what this is doing -- I comment them out and the edit page input titles and fields look the same. You'll see in admin.scss (our customizations) we have 
```
.object fieldset,
.object .field-content {
  width: 100%;
}
```
and page_editor.css (part of wagtail css) there is
```
.object>h2 label {
    display: inline;
    text-transform: inherit;
    font-weight: inherit;
    float: none;
    width: auto;
    color: inherit;
    font-size: inherit;
}
```
2) using !important worries me in that we're making a nightmare for those adjusting styles later. Can we better systematize and scope our style overrides some how so we don't end up writing scss over even our own scss?

Reply:
@ifsimicoded if we're on a large screen we end up with layout issues:
Staging:
<img width="1542" alt="screen shot 2018-10-30 at 12 40 34 pm" src="https://user-images.githubusercontent.com/6384156/47738466-31208580-dc41-11e8-9fc4-2eadc188c85c.png">
This branch:
<img width="1538" alt="screen shot 2018-10-30 at 12 40 59 pm" src="https://user-images.githubusercontent.com/6384156/47738483-367dd000-dc41-11e8-8adc-bbeaeaddf024.png">

I'm not sure how/where to better address this (but I agree that !important is less than ideal)